### PR TITLE
Adapting of business models to UML

### PR DESCRIPTION
**Pull request by Stepa28**

### CODE REVIEW

File: SpaceZD.BusinessLayer/Models/TripModel.cs [hunk 1]
```diff
@@ -1,19 +1,13 @@
...
-namespace SpaceZD.BusinessLayer.Models
+﻿namespace SpaceZD.BusinessLayer.Models
 {
     public class TripModel
     {
         public int Id { get; set; }
         public virtual TrainModel Train { get; set; }
         public virtual RouteModel Route { get; set; }
-        public virtual ICollection<TripStationModel> Stations { get; set; }
+        public virtual LinkedList<TripStationModel> Stations { get; set; }
```
Comment (Negodyaj) [inline SpaceZD.BusinessLayer/Models/TripModel.cs]: please use a List here
and why do we have "virtual" here??

File: SpaceZD.BusinessLayer/Models/TripStationModel.cs [hunk 2]
```diff
@@ -1,10 +1,4 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace SpaceZD.BusinessLayer.Models
+﻿namespace SpaceZD.BusinessLayer.Models
```
Comment (Negodyaj) [inline SpaceZD.BusinessLayer/Models/TripStationModel.cs:1]: get rid of "virtual" modifier

File: SpaceZD.BusinessLayer/Models/TripModel.cs (hunk 1 above)
Comment (Stepa28) [reply SpaceZD.BusinessLayer/Models/TripModel.cs]: поправил

File: SpaceZD.BusinessLayer/Models/TripStationModel.cs (hunk 2 above)
Comment (Stepa28) [reply SpaceZD.BusinessLayer/Models/TripStationModel.cs:1]: поправил

**Pull request merged**